Sources/NIOHTTP2/HTTP2FrameParser.swift

```diff
@@ -685,6 +775,7 @@ struct HTTP2FrameDecoder {
 
     internal var headerDecoder: HPACKDecoder
     private var state: ParserState
+    private var maximumSequentialContinuationFrames: Int
```

Comment:
This should be a `let`. Also we don't need to pass `maximumSequentialContinuationFrames` through all of the states, I think we only need to pass it to the `process` func of the appropriate state.

Reply:
> Also we don't need to pass maximumSequentialContinuationFrames through all of the states, I think we only need to pass it to the process func of the appropriate state.

Yes. I completely missed that. Thanks!

Reply:
This should still be a `let`